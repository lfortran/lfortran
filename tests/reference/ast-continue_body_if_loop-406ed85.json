{
    "basename": "ast-continue_body_if_loop-406ed85",
    "cmd": "lfortran --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/continue_body_if_loop.f",
    "infile_hash": "0b34e65df675e3b4009fe5659fbc93ca52002eb7781599ebb9fe7613",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-continue_body_if_loop-406ed85.stdout",
    "stdout_hash": "76e0f5349eb4e503719ddc923727a2949088c3fe3f8218ae79baa266",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}