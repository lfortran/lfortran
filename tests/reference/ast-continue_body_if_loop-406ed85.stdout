(TranslationUnit [(Program implicit_program_lfortran () [] [] [(Declaration (AttrType TypeReal [] () None) [] [(u [] [] () None ()) (s [] [] () None ())] ()) (Declaration () [(AttrDimension [])] [(u [(1 100 DimensionExpr)] [] () None ())] ()) (Declaration (AttrType TypeInteger [] () None) [] [(j [] [] () None ())] ())] [(= 0 s (Real "0.0") ()) (DoLoop 0 () 1 j 1 100 () [(= 0 s (+ s (FuncCallOrArray u [] [(() j ())] [] [])) ()) (If 0 () (>= s 1000000) [(GoTo 0 () 2 [] ())] [] () () ()) (Continue 1 ())] () ()) (Stop 0 () () ()) (Continue 2 ())] [])])
